hist/hist/src/TH1.cxx

```diff
@@ -6197,6 +6197,26 @@ Bool_t TH1::Multiply(const TH1 *h1, const TH1 *h2, Double_t c1, Double_t c2, Opt
    return kTRUE;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief Normalize a histogram to its maximum value or integral.
+/// @note Works for TH1, TH2, TH3, ...
+/// @param option `"max"`, `"width"` or `""` (default)
+/// If it contains `max`, this histogram is normalized by 1/GetMaximum()
+/// else it is normalized by `1/Integral(option)`, ie it is normalized by `1/sum`
+/// in the default case or by `1/(sum*bin_width)` if option is "width".
+/// In case the norm is zero, it raises an error.
```

Comment:
```suggestion
/// In case the norm is zero, it raises an error.
/// @sa https://root-forum.cern.ch/t/different-ways-of-normalizing-histograms/15582/
```